lib/commands/add.ts

```diff
@@ -94,8 +94,8 @@ command = {
 				cd11: !!config.skipGit,
 				cd14: config.project.theme
 			});
-
-			await command.addTemplate(argv.name, selectedTemplate, argv.module);
+			const selectedModule = argv.module || argv.m;
```

Comment:
This shouldn't be needed - yargs will assign both the alias (`args.m`) and the param (`args.module`) if one is provided.